.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "docs/themes/learn"]
```

Comment:
Do we need this file? The folder seems empty

Reply:
Yes, it's a git submodule. It is pulled automatically by git at checkout.